src/test/ui/error-codes/E0523.rs

```diff
@@ -0,0 +1,9 @@
+// aux-build:e0523_aux_1.rs
+// aux-build:e0523_aux_2.rs
```

Comment:
These two crates both have the same "crate-name", so I was hoping that including both would trigger E0523.

I've tried various combinations of file names and crate names, but so far no luck hitting E0523. As written here, the compiler cannot find the `fifty()` function, which suggests to me that somehoe `*_aux1` is overriding `*_aux2` or something like that.